wajibisha/templates/_footer.html

```diff
@@ -1,38 +1,58 @@
-{% extends '_footer.html' %}
```

Comment:
We're overwriting everything in the footer?

Reply:
We are overriding the default footer section because we are using the same instance as that on hurumap. This prevents us from repeating code that is already in use.